mlflow/types/schema.py

```diff
@@ -119,6 +145,22 @@ def numpy_types(self) -> List[np.dtype]:
         """ Convenience shortcut to get the datatypes as numpy types."""
         return [x.type.to_numpy() for x in self.columns]
 
+    def pandas_types(self) -> List[np.dtype]:
+        """ Convenience shortcut to get the datatypes as pandas types."""
+        return [x.type.to_pandas() for x in self.columns]
+
+    def as_spark_schema(self):
+        """Convert to Spark schema. If this schema is a single unnamed column, it is converted
+        directly the corresponding spark data type, otherwise it's returned as a struct (missing
+        column names are filled with an integer sequence).
+        """
+        if len(self.columns) == 1 and self.columns[0].name is None:
```

Comment:
Do we have test coverage for these various cases?

Reply:
Added tests.